src/Arcus.Templates.Tests.Integration/WebApi/Swagger/v1/SwaggerDocAvailabilityTests.cs

```diff
@@ -21,37 +20,31 @@ public SwaggerDocAvailabilityTests(ITestOutputHelper outputWriter)
             _outputWriter = outputWriter;
         }
 
-        [Theory]
-        [InlineData(BuildConfiguration.Debug, HttpStatusCode.OK)]
-        [InlineData(BuildConfiguration.Release, HttpStatusCode.NotFound)]
-        public async Task GetSwaggerUI_WithBuildConfiguration_Returns(BuildConfiguration buildConfiguration, HttpStatusCode expectedStatusCode)
+        [Fact]
+        public async Task GetSwaggerUI_WithBuildConfiguration_ReturnsOk()
         {
             // Arrange
-            var configuration = TestConfig.Create(buildConfiguration);
-            using (var project = await WebApiProject.StartNewAsync(configuration, _outputWriter)) 
+            using (var project = await WebApiProject.StartNewAsync(_outputWriter)) 
             // Act
             using (HttpResponseMessage response = await project.Swagger.GetSwaggerUIAsync())
             {
                 // Assert
                 Assert.NotNull(response);
-                Assert.Equal(expectedStatusCode, response.StatusCode);
+                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
```

Comment:
Will this work, since I thought we were only going to expose the Open API json document by default, but hide the Swagger UI that shows the openapi documentation ?  Or am i wrong ?

Reply:
Ah, OK, we can do that. Hide the Swagger UI in DEBUG (like before). 👍

Reply:
Ok, done that.